### ISSUE TITLE
Unable to upgrade to 0.2.5

### ISSUE THREAD

**Issue by kramrm** [contentful/rich-text-renderer.py#18]
The following error is given when trying to install v0.2.5 via pip:

```
Collecting rich_text_renderer==0.2.5
  Using cached rich_text_renderer-0.2.5.tar.gz (4.7 kB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [6 lines of output]
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "C:\Users\kramr\user\Local\Temp\pip-install-2sfr66oo\rich-text-renderer_d782b3a986f848dbbc2fb95938f41401\setup.py", line 22, in <module>
          with open('README.rst', 'r') as f:
      FileNotFoundError: [Errno 2] No such file or directory: 'README.rst'
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
```

**Comment by rubydog** [comment]
Hi @kramrm, thanks for reporting the issue. We are taking a look at it.

Cheers

**Comment by stevejalim** [comment]
Hi @rubydog - thanks for the new release, but I'm afraid this is still an ongoing glitch. The README isn't in the collected assets to read:

```
$ pip install rich-text-renderer==0.2.6
Collecting rich-text-renderer==0.2.6
  Using cached rich_text_renderer-0.2.6.tar.gz (4.7 kB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [6 lines of output]
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/private/var/folders/xh/n4lvfgnn48195r6tt0jx_8fh0000gn/T/pip-install-1caexk75/rich-text-renderer_be7363acdbe6420b9dd662a772535620/setup.py", line 63, in <module>
          long_description=open('README.rst').read(),
      FileNotFoundError: [Errno 2] No such file or directory: 'README.rst'
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
```

And the source package (here)[https://files.pythonhosted.org/packages/d1/c5/6d5be555bff6e202205975a642625bb5d9c5b86a44372559e8971a9d4b2f/rich_text_renderer-0.2.6.tar.gz} seems to still lack the README:
```
$ tree rich_text_renderer-0.2.6
rich_text_renderer-0.2.6
├── PKG-INFO
├── rich_text_renderer
│   ├── __init__.py
│   ├── base_node_renderer.py
│   ├── block_renderers.py
│   ├── document_renderers.py
│   ├── null_renderer.py
│   ├── rich_text_renderer.py
│   └── text_renderers.py
└── setup.py
```

I think this issue is worth re-opening.

**Comment by rubydog** [comment]
@stevejalim I agree the issue still exists. The problem was with the old packaging strategy, we have switched to twine now (PR #22). Hopefully, it should be fixed in the next release.

Cheers

**Comment by rubydog** [comment]
Fixed in #22
